pkg/client/interface.go

```diff
@@ -19,24 +19,57 @@ import (
 )
 
 type TxContext interface {
```

Comment:
#PUC for the interface

Reply:
Thanks for calling this out! It left it on the wrong branch. :sweat_smile: